### ISSUE TITLE
Telegram Support

### ISSUE THREAD

**Issue by cj-sv** [system76/pop-gtk-theme#57]
Telegram support would be awesome. Adapta-GTK comes with a Telegram theme. We could probably base it off of that.

https://github.com/adapta-project/adapta-gtk-theme/tree/master/extra/telegram

**Comment by isantop** [comment]
I agree, that would be nice. Since Telegram doesn't pick up any GTK theming and it's not something that gets shipped by default, we'd likely be looking for input from a community member to maintain the theme.

**Comment by cj-sv** [comment]
Awesome. Would it be possible to release the color palette (on readme maybe)? That would make things easier.

**Comment by isantop** [comment]
The full color palette is stored in the theme SCSS. https://github.com/system76/pop-gtk-theme/blob/master/src/gtk-3.0/3.22/sass/_color-palette.scss

Please keep in mind that only the first 41 lines describe actual colors we use.

**Comment by cj-sv** [comment]
Made a temporary solution based off of DarkOrange v2. Some highlights and accents may have been used wrong. Up for improvement.

![screenshot from 2017-10-17 14-47-35](https://user-images.githubusercontent.com/11567756/31651197-47ca809c-b34d-11e7-8649-00cdd8bec68e.png)


![screenshot from 2017-10-17 14-47-41](https://user-images.githubusercontent.com/11567756/31651195-467091fa-b34d-11e7-8455-ec1210de71cd.png)

[pop-dark-telegram.tar.gz](https://github.com/system76/pop-gtk-theme/files/1390135/pop-dark-telegram.tar.gz)

**Comment by isantop** [comment]
Couple of notes:

The orange should be #cc7900. That should precisely match the orange used in the Pop Dark theme. Also, the selected item in the menu should have a background of the orange, with white content.

**Comment by pedrofleck** [comment]
I've been making this theme for some time and I think it is ready for testing! If someone else try it, give me some feedback and hopefully we can bump it to version 1.0 :)
[Link](https://t.me/addtheme/rZceTfSsyGuiUfeL) to install the theme directly.
[Popgram 0.3.zip](https://github.com/pop-os/gtk-theme/files/3769852/Popgram.0.3.zip)
